craterpy/tests/test_helper.py

```diff
@@ -10,30 +10,6 @@
 class Test_geo_helpers(unittest.TestCase):
     """Test Geospatial Helper Functions"""
 
-    def test_lon360(self):
-        """Test lon360 function"""
-        actual = ch.lon360(0)
-        expected = 0
-        self.assertEqual(actual, expected)
-        actual = ch.lon360(360)
-        expected = 0
-        self.assertEqual(actual, expected)
-        actual = ch.lon360(20)
-        expected = 20
-        self.assertEqual(actual, expected)
-        actual = ch.lon360(185)
-        expected = 185
-        self.assertEqual(actual, expected)
-        actual = ch.lon360(355)
-        expected = 355
-        self.assertEqual(actual, expected)
-        actual = ch.lon360(-175)
-        expected = 185
-        self.assertEqual(actual, expected)
-        actual = ch.lon360(-5)
-        expected = 355
-        self.assertEqual(actual, expected)
-
```

Comment:
lol this test is really funny to me in hindsight. good example of overtesting. We just need to see that the edge cases and different types of inputs work. here we only test int and it's unclear what e.g. 20 reveals that other inputs wouldn't already show